meta-mender-qemu/recipes-core/systemd/files/01-logs-to-console.conf

```diff
@@ -0,0 +1,2 @@
+[Journal]
+ForwardToConsole=true
```

Comment:
Ok, let's drop it for now an figure out a way to add it the same way we modify tokens/server url and such.